src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -191,18 +191,14 @@ class GroupEventDetails extends React.Component<Props, State> {
                 location={location}
               />
             )}
-            {group.status !== 'unresolved' && (
-              <div className="issue-status">
-                {group.status === 'ignored' && (
-                  <MutedBox statusDetails={group.statusDetails} />
-                )}
-                {group.status === 'resolved' && (
-                  <ResolutionBox
-                    statusDetails={group.statusDetails}
-                    orgId={organization.slug}
-                  />
-                )}
-              </div>
+            {group.status === 'ignored' && (
```

Comment:
I know that this is probably out of scope of this PR, but should we make enum out of these?

Reply:
An enum for these could be helpful.